src/components/Advisor/Advisor.js

```diff
@@ -32,16 +32,7 @@ const useStyles = makeStyles(theme => ({
     justifyContent: "flex-end"
   },
   next: {
-    alignitems: "right"
-  },
-  cellNext: {
-    backgroundColor: "white",
-    color: "#A41E35",
-    cursor: "pointer",
-    "&:hover": {
```

Comment:
This styling for cellNext should still be here from the master branch. Seems like the same issue with pulling master from before when the AnimatedModal still existed.